bin/inasafe.py

```diff
@@ -32,11 +33,15 @@
     QgsMapLayerRegistry,
     QgsCoordinateReferenceSystem)
 
+
+from safe.test.utilities import get_qgis_app
+# make sure this line executes first
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
```

Comment:
Yeah we should probably not be importing test code into production. Conceivably we could be stripping test code out of production shipped packages.